code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -162,3 +162,18 @@
 		randname += " [lastname]"
 
 	return randname
+
+/datum/species/plasmaman/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+    . = ..()
+
+    if(chem.type == /datum/reagent/toxin/plasma)
+        H.adjustOxyLoss(-1)
+        H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
```

Comment:
```suggestion
    if(chem.type == /datum/reagent/toxin/plasma)
        H.adjustOxyLoss(-1)
        H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
        return TRUE
```
Otherwise it will start doing what plasma normally does

Reply:
Done.